code/modules/surgery/surgery.dm

```diff
@@ -141,6 +147,86 @@
 	SSblackbox.record_feedback("tally", "surgeries_completed", 1, type)
 	qdel(src)
 
+/datum/surgery/proc/on_mouse_entered(datum/_source, mob/living/user)
+	SIGNAL_HANDLER
+	if(!istype(user) || user.zone_selected != location || !(HAS_TRAIT(user, TRAIT_MEDICAL_HUD) || HAS_TRAIT(user, TRAIT_SURGEON) || HAS_TRAIT(user, TRAIT_ABDUCTOR_SURGEON)) || (user in hovering_users))
```

Comment:
If i remember correctly, abductors get the surgeon trait for their mind and not the mob, so should be `user.mind` instead
or something like this if one wants to have both
`if(HAS_TRAIT(user, TRAIT_ABDUCTOR_SURGEON) || (user.mind && HAS_TRAIT(user.mind, TRAIT_ABDUCTOR_SURGEON)))`